src/Label.tsx

```diff
@@ -70,7 +70,7 @@ const Label = styled.span<
 
 Label.defaultProps = {
   theme,
-  bg: 'gray.5',
+  bg: get('colors.label.primary.border'),
```

Comment:
I don't think this is the right way to do this - is there a correct way to reference these color vars outside of a template?

Reply:
These props are theme-aware. So you can pass the theme key directly. Props like `bg` and `color` assume you're indexing into the `colors` key so you don't need to write it:

```suggestion
  bg: 'label.primary.border',
```

Reply:
ahhh, okay, that makes sense!